uefi/flash.go

```diff
@@ -26,6 +27,18 @@ type FlashImage struct {
 	BiosRegion *BiosRegion
 }
 
+type jsonFlashImage FlashImage
+
+func (f FlashImage) MarshalJSON() ([]byte, error) {
+	return json.Marshal(struct {
+		Size int
```

Comment:
Why the size? Unnecessary afaict

Reply:
just json.Marshal(f) should work?

Reply:
Size is nice for looking at the summary, I can get rid of it. If I don't need the size then I don't need all this stuff.